client/src/main/java/no/nav/common/client/axsys/AxsysClient.java

```diff
@@ -0,0 +1,13 @@
+package no.nav.common.client.axsys;
+
+import no.nav.common.health.HealthCheck;
+import no.nav.common.types.identer.EnhetId;
+import no.nav.common.types.identer.NavIdent;
+
+import java.util.List;
+
+public interface AxsysClient extends HealthCheck {
+    List<NavIdent> hentAnsatte(EnhetId enhetId);
+
+    AxsysEnheter hentTilganger(NavIdent veileder);
```

Comment:
Tenker at det er bedre å returnere `List<AxsysEnhet>`. Da må det også flyttes litt på data klassen slik at AxsysEnhet får en egen fil, så kan du bruke en anonym innerklasse for AxsysEnheter.
Litt småpirk, men det burde også stå `NavIdent navIdent`